CHANGELOG.md

```diff
@@ -1 +1,14 @@
 # Changelog
+
+## [0.2.2](https://github.com/nodejs/amaro/compare/v0.2.1...v0.2.2) (2025-01-06)
+
+
+### Miscellaneous
+
+* build wasm from swc v1.10.4 ([a00240a](https://github.com/nodejs/amaro/commit/a00240acbebce535144b26b6ebb6bc65c7e04d8e))
+* **deps:** bump github/codeql-action from 3.27.0 to 3.28.0 ([88051e0](https://github.com/nodejs/amaro/commit/88051e07265c4f33dd6b0e55e73df176492b1236))
+* include chore in change logs ([9ab6341](https://github.com/nodejs/amaro/commit/9ab6341c697116fa4f5af656a1390c4f228531b9))
+* update release actions ([9538187](https://github.com/nodejs/amaro/commit/95381873d9dd2fbe720ab01138b7653e3afc12d4))
+* update swc to v1.10.4 ([af1fd9c](https://github.com/nodejs/amaro/commit/af1fd9c3aa795bc9e0adb3907662e8b254899b57))
+
+## Changelog
```

Comment:
why the  ## changelog section is empty?

Reply:
I think it is an issue with the initial changelog generation. Once I added something (e.g. https://github.com/legendecas/amaro/pull/3/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL3), it works well. I'm going to edit this manually.